docs/specifications/tests/DNSSEC-TP/dnssec15.md

```diff
@@ -0,0 +1,176 @@
+# DNSSEC15: Existence of CDS and CDNSKEY
+
+## Test case identifier
+**DNSSEC15**
+
+## Objective
+
+CDS and CDNSKEY record types are defined in [RFC 7344] and [RFC 8878].
+Both record types are optional in a zone. The objective of this test
+case is to verify that they are correctly set-up, if included in the
+zone.
+
+If an CDS record is included in the zone, the corresponding CDNSKEY
```

Comment:
If a CDS record is included in the zone, the corresponding CDNSKEy

Reply:
Fixed.